src/main/kubectl.ts

```diff
@@ -58,7 +64,6 @@ export class Kubectl {
     return path.join((app || remote.app).getPath("userData"), "binaries", "kubectl")
   }
 
-  public static readonly bundledKubectlPath = bundledPath
   public static readonly bundledKubectlVersion: string = bundledVersion
   public static invalidBundle = false
   private static bundledInstance: Kubectl;
```

Comment:
I cannot review line 72 but should it be `public stattic get bundled() { ... }`?

Reply:
Not sure what you mean 🤔

Reply:
I think I meant that I thought it would be better if the way to get the bundled kube instance was through a static getter instead of a static function